changelogs/changelog.yaml

```diff
@@ -6,3 +6,10 @@ releases:
     release_date: '2021-02-11'
   0.3.0:
     release_date: '2021-04-06'
+  1.0.0:
+    changes:
+      release_summary: 'This is the first stable release of the ``theforeman.operations``
+        collection.
+
+        '
```

Comment:
This block of text here looks weird to me

Reply:
that's how `antsibul-changelog` generates it. :woman_shrugging:

Reply:
(fixed)